lib/errorTypes.js

```diff
@@ -3,4 +3,4 @@
 var errors = require('restify-errors');
 
 errors.makeConstructor('RequestCloseError');
-errors.makeConstructor('RequestAbortedError');
+errors.makeConstructor('RouteMissingError');
```

Comment:
Where is this error used?

Reply:
This error existed before but was created locally.
When a code calls `next('route-name')` but the route doesn't exist.
It's a programmatic error.